### ISSUE TITLE
ARM AArch64 support

### ISSUE THREAD

**Issue by ncying** [virtualeconomy/v-systems#167]
In current code, we used the latest released version of `leveldbjni` (leveldbjni-1.8, on 17 Oct 2013, https://github.com/fusesource/leveldbjni), this may not support the Android Phone to run a full-node. 

In their latest merged PR (on 2 Nov 2017, https://github.com/fusesource/leveldbjni/pull/82), ARM AArch64 is supported. We may add this feature to our project.
```
Added ARM AArch64 support on Linux.
Build and test are passed by issuing:
mvn clean package -P download -P linux64-aarch64
```

Tasks:
1. May fork the repo to our project
2. compile and build the code
3. release the package to mvn
4. update the package information in https://github.com/virtualeconomy/v-systems/ https://github.com/virtualeconomy/v-systems/blob/0b5ba8ddab5e072d9f09c923aacccf8b2bdca6ef/project/Dependencies.scala#L45

**Comment by faddat** [comment]
Is this task complete?

Sounds like something I'd enjoy.

What did you mean by `may fork the repo to our project`?

**Comment by ncying** [comment]
No. This is a task in the plan. Some particular packages should be revised to satisfy Linux in mobile phones. Then the user can run a node in mobile phone.

I mean it should fork the package repo and revised by us

**Comment by faddat** [comment]
Does the host system need leveldb?

I think so because I have run VSYS without it, but I am not sure. Thanks for any info!

**Comment by faddat** [comment]
https://mvnrepository.com/artifact/org.openlabtesting.leveldbjni/leveldbjni-linux64-aarch64/1.8

I'm trying this now but it is architecture specific so maybe not the best possible choice

**Comment by faddat** [comment]
boom, vsys on pi:

<img width="1039" alt="Screen Shot 2020-09-09 at 2 32 46 PM" src="https://user-images.githubusercontent.com/7142025/92568438-5fddfd00-f2a9-11ea-88a2-2164991e689c.png">

**Comment by ncying** [comment]
I checked several packages that can make this work on my android phone, but I am not sure about the security problems back the packages.

We should take care to use such dependency for the official release, and listed choices to the users.

**Comment by faddat** [comment]
Closed by #229 

The package I used is based on the master branch of the package that we are currently using and I do not see any security risk.
